aws/solutions/TaggingRootVolumesInEC2/LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+MIT No Attribution

```

Comment:
This repo uses the Apache-2.0 license. Please remove the MIT license from this directory.